src/components/jnlp-page/Hero-JNLP.astro

```diff
@@ -0,0 +1,67 @@
+---
+import LinkButton from "../LinkButton.astro";
+import { Image } from "astro:assets";
+import { DISCORD_URL } from "../../consts";
+import Icon from "astro-icon";
+import JNLPBox from "./assets/box-JNLP.png";
+import sparkleIcon from "./assets/sparkle.png";
+---
+
+<header
+	class="min-h-[calc(100vh-48px)] flex flex-col items-center bg-gradient-to-b to-[#e6275544] from-stone-950"
+>
+	<div class="grow"></div>
+
+	<div
+		class="w-full max-w-screen-xl px-4 sm:px-8 py-16 space-y-4 md:grid grid-cols-3"
+	>
+		<div class="flex items-center col-span-2">
+			<div>
+				<h1
+					class="text-white text-4xl md:text-5xl lg:text-6xl xl:text-7xl font-semibold"
+				>
+					Run
+					<span class="font-bold text-primary-500"> Java Web Start</span>
+					<br />apps in the browser
+				</h1>
+				<p class="text-xl mt-8 text-stone-400 leading-6">
+					Unleash the power of CheerpJ now available in a easy-to-use browser
+					extension. Run Java Web Start Applications, including Oracle
+					E-Business Suite and Oracle Forms Applications, without a Java Desktop
+					installation.
```

Comment:
Maybe have the second sentence in its own paragraph? It's a fairly big block of 3 lines of text at the moment.

That is

```
Unleash the power of CheerpJ, now available in a easy-to-use browser extension!

Run Java Web Start Applications, including Oracle E-Business Suite and
Oracle Forms Applications, without a Java Desktop installation.
```

If it looks OK, we should also consider using non breaking spaces (`&nbsp;`) instead of normal spaces in phrases like "Java Web Start Applications". This will prevent reflowing like

```
Run Java Web Start Applications, including Oracle
E-Business Suite and Oracle Forms
Applications, without a Java
Desktop installation.
```

which is hard to parse imo.